model/websocket_message.go

```diff
@@ -74,57 +74,125 @@ type precomputedWebSocketEventJSON struct {
 	Broadcast json.RawMessage
 }
 
-type WebSocketEvent struct {
```

Comment:
Even tough I think these changes are very valuable, they are breaking semver and hence we can't do this without releasing a new major version.

Reply:
Thank you @hanzei , that's a good point. The type name itself did not change, nor the existing functions but I agree that making the fields unexported and replacing them with getters can be considered a breaking change. Not sure there's a robust way to fix this issue without making that type [opaque](https://en.wikipedia.org/wiki/Opaque_data_type).

Reply:
I can't think of a way either. I would opt for exporting the fields, but marking them as deprecated and recommend to use the Getter.

Reply:
Yep, let's export them and mark as deprecated. Great work @streamer45

Reply:
Sounds good to me. @jespino, @jwilander thoughts?

Reply:
And also to add a comment to unexport the fields when 6.x comes out.

Reply:
Yes, I think is enough to have them published, my concern would be around miss-usage of them generating race conditions, but I guess it was racy before.